doc/classes/Rect2.xml

```diff
@@ -50,63 +50,83 @@
 			<param index="2" name="width" type="float" />
 			<param index="3" name="height" type="float" />
 			<description>
-				Constructs a [Rect2] by x, y, width, and height.
+				Constructs a [Rect2] by setting its [member position] to [param x] and [param y], and its [member size] to [param width] and [param height].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="abs" qualifiers="const">
 			<return type="Rect2" />
 			<description>
-				Returns a [Rect2] with equivalent position and area, modified so that the top-left corner is the origin and [code]width[/code] and [code]height[/code] are positive.
+				Returns a [Rect2] equivalent to this rectangle, with its width and height modified to be positive values, and its [member position] turned into the top-left corner of the rectangle.
+				[codeblocks]
+				[gdscript]
+				var rect = Rect2(25, 25, -100, -50)
+				var absolute = rect.abs() # absolute is Rect2(-75, -25, 100, 50)
+				[/gdscript]
+				[csharp]
+				var rect = new Rect2(25, 25, -100, -50);
+				var absolute = rect.Abs(); // absolute is Rect2(-75, -25, 100, 50)
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] It's recommended to use this method when [member size] is negative, as most other methods in Godot assume that the [member position] is the top-left corner, and the [member end] is the bottom-right corner.
 			</description>
 		</method>
 		<method name="encloses" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="b" type="Rect2" />
 			<description>
-				Returns [code]true[/code] if this [Rect2] completely encloses another one.
+				Returns [code]true[/code] if this rectangle [i]completely[/i] encloses the [param b] rectangle.
 			</description>
 		</method>
 		<method name="expand" qualifiers="const">
 			<return type="Rect2" />
 			<param index="0" name="to" type="Vector2" />
 			<description>
-				Returns a copy of this [Rect2] expanded to include a given point.
-				[b]Example:[/b]
+				Expands this rectangle to align the given [param to] point around the edges, if necessary.
```

Comment:
The previous wording "Returns a copy of" was better, because the new formulation could be interpreted as "expand" changes the `Rect2`.

Reply:
A lot of methods that return a modification of their value are easier to imagine when explained this way (e.g. _"Strips all escape characters from the string"_ instead of _"Returns a copy of the string stripped of all escape characters"_.

However, I feel some sort of dejavu, like this issue has already popped up in the past. The new description is oddly verbose

Reply:
Those cases should honestly be fixed to clarify copy, it's an important distinction and users aren't guaranteed to assume that the method having `const` means this